model-optimizer/extensions/back/SpecialNodesFinalization.py

```diff
@@ -126,26 +126,24 @@ class RemoveConstToResult(BackReplacementPattern):
     def pattern():
         return dict(
             nodes=[
-                ('const_node', {'type': 'Const', 'kind': 'op'}),
-                ('const_data', {'kind': 'data'}),
+                ('const_data', {'kind': 'data', 'value': lambda value: value is not None}),
```

Comment:
I am afraid that it will break e2e test for W&D since this transformation may no longer apply for it due to `lambda` in condition. @jane-intel, could you please check?

Reply:
This whole PR is for W&D. So it works

Reply:
Great!